app/src/main/java/com/example/todo/ListActivity.java

```diff
@@ -209,6 +208,37 @@ public void change_theme_popup() {
         alertDialog.show();
     }
 
+    public void delete_list_popup(final int lID){
```

Comment:
Please use meaningful names for variables. And in case a global variable has the same name already, Java provides you with the ability to refer to the local variable using _this_ keyword. And hence the same name can be used for both.

Here, in this case, take a look if you can make the same function work without passing the parameter altogether.

Reply:
Thanks  for guiding! Actually there was no need for a local variable. I referred the usages in change_theme function and have modified the code in my recent commit.